### PR TITLE
add warning about uninstall with filename

### PR DESCRIPTION
**Pull request by richardwxn**
This is applicable to `istioctl x uninstall` with filename flag. Resolves: https://github.com/istio/istio.io/issues/7927. The actual fix is: https://github.com/istio/istio/pull/26449

### CODE REVIEW

File: content/en/docs/setup/upgrade/index.md [hunk 1]
```diff
@@ -108,6 +108,10 @@ If the old control plane does not have a revision label, uninstall it using its
 $ istioctl x uninstall -f manifests/profiles/default.yaml
 {{< /text >}}
 
+{{< warning >}}
+Note that this command would also remove gateway deployments, use it with caution. The issue would be fixed in release `1.7.1`
```
Comment (frankbu) [inline content/en/docs/setup/upgrade/index.md:112]:
```suggestion
This command will also remove gateway deployments, which will then need to be redeployed. The issue will be fixed in release `1.7.1`
```

Comment (linsun) [reply content/en/docs/setup/upgrade/index.md:112]: @richardwxn if the fix is NOT in 1.7.0, can we just recommend users how to clean up manually without uninstall cmd?

Using uninstall cmd would create a non working cluster for users without the fix.

Comment (richardwxn) [reply content/en/docs/setup/upgrade/index.md:112]: @linsun the use cases of --revision and --purge works, which I expect most people would use, I can remove the part of -f, wdyt?

Comment (linsun) [reply content/en/docs/setup/upgrade/index.md:112]: Looks like the fix will land in 1.7.0 afterall.  If --revision works for default, I agree with removing the -f portion.

Comment (richardwxn) [reply content/en/docs/setup/upgrade/index.md]: updated the description

File: content/en/docs/setup/upgrade/index.md [hunk 2]
```diff
@@ -102,6 +102,10 @@ To uninstall a control plane of revision `istio-1-6-5`, run:
 $ istioctl x uninstall --revision istio-1-6-5
 {{< /text >}}
 
+{{< warning >}}
+This command will also remove gateway deployments for specified revision. If you want to remove the canary revision, please reinstall the gateway(s) with previous revision first to avoid down time.
```
Comment (linsun) [inline content/en/docs/setup/upgrade/index.md:106]: Is this correct all the time?  I think it won't remove if I'm deleting the old revision (1-6-5) but would if i'm deleting the new revision (1-7-0).

Comment (linsun) [reply content/en/docs/setup/upgrade/index.md:106]: I propose to have 2 sections, 1 for remove old cp, 1 for remove the new version (canary).  This warning only applied to the 2nd scenario.

Comment (frankbu) [reply content/en/docs/setup/upgrade/index.md:106]: I added some text for this along with another fix in #7957. If that looks alright, then we can close this PR.

Comment (richardwxn) [reply content/en/docs/setup/upgrade/index.md:106]: @linsun yes, that's why I emphasized `canary` one here. But if that's not clear enough, then different section is ok

**Comment by frankbu** [comment]
Closing, because this is addressed in #7957. Please reopen or send a followup PR if more changes are needed.

**Pull request closed without merging**